integration-tests/vm.install

```diff
@@ -12,7 +12,7 @@ tar -xzf subscription-manager.tar.gz --transform 's,[^/]*,/sm,'
 cd sm
 tar xzf ../subscription-manager-build-extra.tar.gz
 python3 ./setup.py build
-python3 ./setup.py install --prefix /usr --root /
+RPM_BUILD_ROOT="" python3 ./setup.py install --prefix /usr --root /
```

Comment:
As for the workaround itself, I suggest the following:

```suggestion
# Simulate installation via RPM package by setting $RPM_BUILD_ROOT
# That way, the RPM-specific installation scheme is selected in Fedora 36+
RPM_BUILD_ROOT="/" python3 ./setup.py install
```

Note three things:

 - the intent of setting `$RPM_BUILD_ROOT` is visible
 - the actual value of `$RPM_BUILD_ROOT` actually make sense (while technically, it doesn't matter)
 - the `--prefix` and `--root` arguments are probably useless so I've removed them

Reply:
> as they are needed in older Fedora and RHEL version

Are you sure?

Reply:
Root always defaults to /. And prefix defaults to `/usr` when `$RPM_BUILD_ROOT` is set.

Reply:
I need `--root`, as at least on RHEL 9 it did change the behaviour; see commit c7d74b75bdbd55cd1d1f760a894f154fbf293c1c (from #2840).

Reply:
Mea culpa.  I've just tested this on CentOS Stream 8 and there is indeed a very weird error happening without `--root`:

```
running install_data
error: [Errno 2] No such file or directory: 'build/locale'
```

With `--root /`, it is gone. The reason appears to be that without `--root` the build process goes through `bdist_egg` and the build fails as your custom build class does not support that. It appears the `--root` option [serves as another undocumented workaround](https://stackoverflow.com/a/33791008/1839451) here. Oh my.

Reply:
I consider my concerns resolved. Note however that IMHO you are piling up workarounds on top of workarounds and actually building the RPM might be safer.

Reply:
Well, the bigger workaround here is the sucky buildsystem, so there isn't much "good" doable when the baseline is already poor... :-(